src/2_RTB/VerbaliEsterni/VerbaleEsterno_061223/content.typ

```diff
@@ -0,0 +1,83 @@
+
+
+== FeedBack della proponente
+
+La Proponente ha sollevato una critica sostanziale in merito alla comprensibilità della dashboard, evidenziando la difficoltà di interpretare il grafico sinusoidale a causa della sovrapposizione di dati. La soluzione proposta è quella di rendere le temperature più singolari e differenziare il modo in cui i sensori generino l'onda sinusoidale, accelerando il ciclo per rappresentare una "giornata" in pochi minuti e aumentare il tasso di generazione dei dati per una rappresentazione più dinamica, inoltre è stato consigliato di aggiungere un pannello descrittivo.
+
+Per ottimizzare le query, è stato suggerito di integrare, ove necessario, le funzioni aggregate di movingAverage e di ridurre l'intervallo per una maggior visuale.
+
+Inoltre, è stata proposta l'implementazione della capacità di filtraggio in Grafana, focalizzata sulla possibilità di effettuare confronti tra sensori.
+
+
+== Obiettivo del prossimo sprint     
+
+La Proponente ha mostrato un forte entusiasmo per i progressi raggiunti in poche settimane, indicando che, dal loro punto di vista, si era già pronti per il PoC. In vista del prossimo sprint, hanno suggerito alcune potenziali migliorie estetiche per rendere più comprensibile il prodotto e la possiblità di aggiungere già un altra tipologia di sensore.
+
+Gli obbiettivi da raggiungere:
+
+  - Differenziazione delle onde sinusoidali dei vari sensori;
+  - Accelerazione del ciclo rappresentativo della giornata;
+  - Aggiunta di un pannello descrittivo;
+  - Rinomina del nome dei sensori;
+  - Integrazione delle funzioni movingAverage di Clickhouse;
+  - Aggiunta di un nuovo tipo di sensore.
+
+
+== Documentazione  
+
+La proponente risultava essere soddisfatta anche della nuova versione dell'Analisi dei Requisiti, l'unica perplessità riguardava il numero complessivo della tipoliogia di sensori, ritenuto eccessivamente elevato anche in termini di visualizzazione sulla dashboard, e alcuni casi d'uso considerati ridondanti. 
+Da notare che la proponente ha sottolineato la preferenza per la qualità rispetto alla quantità. In altre parole, si è manifestato un interesse più marcato verso sensori altamente efficaci e pertinenti piuttosto che un vasto assortimento che potrebbe risultare difficile da gestire o che include elementi superflui.
+La proponente ha anche sottolineato che l'individuazione dei casi d'uso risulta essere più complessa a causa del fatto che il sistema è quasi esclusivamente backend. In questo contesto, la natura prevalentemente di backend potrebbe rendere più sfumata la comprensione e l'identificazione dei casi d'uso
```

Comment:
In questo contesto, la natura prevalentemente di backend potrebbe rendere più sfumata la comprensione e l'identificazione dei casi d'uso


potrebbe rendere più sfumata la comprensione?